certgen/certgen.go

```diff
@@ -83,6 +83,43 @@ type CA struct {
 // in the SigningCert and SigningKey fields of the CA, and configured using the
 // CertConfig in the CA.
 func (ca *CA) IssueCertificate() (*x509.Certificate, error) {
+	return ca.issueCertificate(false)
+}
+
+// IssueCertificateChain creates a new certificate chain, containing a new leaf
```

Comment:
new twice; maybe ding the first? It's implicitly a new chain but it's worth saying the leaf itself is new.